samples/bluetooth/direct_test_mode/README.rst

```diff
@@ -278,6 +271,123 @@ Testing with nRF Connect for Desktop
 #. Swap roles.
    Set the application to the RX mode and the connected development kit to the TX mode.
 
+Direct Test Mode terminal connection
+------------------------------------
+
+To run Direct Test Mode (DTM) to send commands and receive responses, you can connect to your development kit with RealTerm in Windows or Minicom in Linux.
```

Comment:
Can you run DTM? This sounds strange to me.
Maybe "To use the Direct Test Mode sample to send ..." or "To send commands to and receive responses from the development kit that runs the Direct Test Mode sample, connect to it with ..."

Reply:
Yes you right. This sounds a little strange. I used your second suggestion.